clients/vault/src/oracle/collector/proof_builder.rs

```diff
@@ -42,17 +43,36 @@ pub enum ProofStatus {
 	NoTxSetFound(Slot),
 }
 
+const MAX_SLOT_TO_REMEMBER : u64 = 12;
```

Comment:
Maybe move this to `clients/vault/src/oracle/constants.rs`

Reply:
Done.